### PR TITLE
[do-not-merge]Dummy pr for ci of ocis latest bump

### PR DESCRIPTION
**Pull request by S-Panta**

### CODE REVIEW

File: tests/integration/compose.yaml [hunk 1]
```diff
@@ -63,6 +63,8 @@ services:
       IDM_ADMIN_PASSWORD: admin
       PROXY_AUTOPROVISION_ACCOUNTS: true
       PROXY_ROLE_ASSIGNMENT_DRIVER: oidc
+      MICRO_REGISTRY: natsjs
+      MICRO_REGISTRY_ADDRESS: 127.0.0.1:9233
```
Comment (saw-jan) [inline tests/integration/compose.yaml:67]:
```suggestion
      MICRO_REGISTRY_ADDRESS: 0.0.0.0:9233
```

Comment (saw-jan) [reply tests/integration/compose.yaml:67]: but it's just for a while. fix should arrive in ocis soon

Comment (amrita-shrestha) [reply tests/integration/compose.yaml:67]: This PR just to check changes in share api

Comment (saw-jan) [reply tests/integration/compose.yaml:67]: may be wait until PR https://github.com/owncloud/ocis/pull/8028 is merged?

Comment (saw-jan) [reply tests/integration/compose.yaml:67]: Ahh, okey

Comment (amrita-shrestha) [reply tests/integration/compose.yaml:67]: we will not merge this PR, we just want to check share api current behavior

**Pull request closed without merging**